bentoml/_internal/cli/bento_management.py

```diff
@@ -170,35 +174,56 @@ def delete_target(target):
 
                 if delete_confirmed:
                     bento_store.delete(bento.tag)
-                    click.echo(f"{bento} deleted")
+                    logger.info(f"{bento} deleted")
 
         for target in delete_targets:
             delete_target(target)
 
-    @cli.command(help="Export Bento to a tar file")
+    @cli.command()
```

Comment:
why we delete the help message here?

Reply:
in `_save()` we have the same message so it will display twice

Reply:
@aarnphm to let click use docstring to  display help message. The docstring is more detailed.

Reply:
i see